imports/api/applicants.js

```diff
@@ -2,6 +2,54 @@ import { Mongo } from 'meteor/mongo';
  
 export const Applicants = new Mongo.Collection('applicants');
 
```

Comment:
Code for uploading CSV came from https://themeteorchef.com/snippets/importing-csvs/